src/dq_suite/output_transformations.py

```diff
@@ -316,7 +319,7 @@ def get_single_validation_result_dict(
         "dqDatum": run_time,
         # TODO/check: rename dqDatum, discuss all field names
         "dqResultaat": validation_result,
-        "regelNaam": expectation_result["expectation_type"],
+        "regelNaam": humps.pascalize(expectation_result["expectation_type"]),
```

Comment:
Hier eigenlijk dezelfde vraag... 

Ook al hangt dit af van de GX internals, en zou me kunnen voorstellen dat GX deze naam misschien in ander formaat teruggeeft, maar ben wel benieuwd.